server/src/main/java/org/elasticsearch/discovery/Discovery.java

```diff
@@ -48,6 +49,7 @@
     void publish(ClusterChangedEvent clusterChangedEvent, AckListener ackListener);
 
     interface AckListener {
+        void onCommit(TimeValue commitTime);
```

Comment:
java docs please on what the commitTime means/

Reply:
fixed in 087518e1cf3